modules/giapi/src/main/scala/giapi/client/GiapiConfig.scala

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package giapi.client
+
+import cats.Show
+
+// Produce a configuration string for Giapi.
+trait GiapiConfig[T] {
+  def configuration(t: T): String
```

Comment:
Maybe `configurationValue` or `configValue` could be more explicit?

Reply:
Sure. I think that sounds better.